deny.toml

```diff
@@ -120,6 +137,7 @@ confidence-threshold = 0.8
 # aren't accepted for every possible crate as with the normal allow list
 exceptions = [
     # Each entry is the crate and version constraint, and its specific allow list.
+    { name = "number_range", version = "0.3.2", allow = ["GPL-3.0"] }
```

Comment:
Please link to https://github.com/aptos-labs/aptos-core/issues/13931: we should be able to remove this as soon as we upgrade the aptos-node fork